### DISCUSSION TITLE
What does "Valar morghulis" mean?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#848] [Galaxy Trivia] [answered]
**Bonus:** What to reply?

**Comment by peterbaccaro** [comment, marked as answer]
"Valar morghulis" is a phrase from the fantasy novel series "A Song of Ice and Fire" by George R.R. Martin and its television adaptation, "Game of Thrones." The phrase is in the fictional High Valyrian language and is commonly translated as "**All men must die**".

In the context of the story, "Valar morghulis" is a common saying in Essos, one of the continents in the world of "A Song of Ice and Fire". It is often used as a reminder of mortality and the belief that death is an inevitable part of life. The phrase is frequently answered with "**Valar dohaeris**", which translates to "All men must serve," suggesting that regardless of one's position or status, everyone has a role to play in the world.
